scripts/coinweight.lua

```diff
@@ -57,10 +59,12 @@ local function createCoinsItem(nodeChar)
 end
 
 ---	This function looks for the "Coins" inventory item if it already exists.
+---	Also matches "Coins (Coins Weight Extension)" for more context in name.
 local function findCoinsItem(nodeChar)
 	for _,nodeItem in pairs(DB.getChildren(nodeChar, 'inventorylist')) do
 		local sItemName = DB.getValue(nodeItem, 'name', '')
-		if sItemName == 'Coins' then
+		if sItemName == COINS_INVENTORY_ITEM_NAME
```

Comment:
Have the initialization of the item and the matcher use the same constant value.